openfoodfacts_exports/exports/csv_export/csv_from_parquet.py

```diff
@@ -0,0 +1,1045 @@
+                ),
+                return_dtype=pl.Float32,
+            ),
+            pl.col("nutriments")
+            .alias("omega-3-fat_100g")
+            .map_elements(
+                lambda x: process_nutrients(nutrients=x, nutrient_name="omega-3-fat"),
+                return_dtype=pl.Float32,
+            ),
+            pl.col("nutriments")
+            .alias("omega-6-fat_100g")
+            .map_elements(
+                lambda x: process_nutrients(nutrients=x, nutrient_name="omega-6-fat"),
+                return_dtype=pl.Float32,
+            ),
+            pl.col("nutriments")
+            .alias("alpha-linolenic-acid_100g")
+            .map_elements(
+                lambda x: process_nutrients(
+                    nutrients=x, nutrient_name="alpha-linolenic-acid"
+                ),
+                return_dtype=pl.Float32,
+            ),
+            pl.col("nutriments")
+            .alias("eicosapentaenoic-acid_100g")
+            .map_elements(
+                lambda x: process_nutrients(
+                    nutrients=x, nutrient_name="eicosapentaenoic-acid"
+                ),
+                return_dtype=pl.Float32,
+            ),
+            pl.col("nutriments")
+            .alias("docosahexaenoic-acid_100g")
+            .map_elements(
+                lambda x: process_nutrients(
+                    nutrients=x, nutrient_name="docosahexaenoic-acid"
+                ),
+                return_dtype=pl.Float32,
+            ),
+            pl.col("nutriments")
+            .alias("linoleic-acid_100g")
+            .map_elements(
+                lambda x: process_nutrients(nutrients=x, nutrient_name="linoleic-acid"),
+                return_dtype=pl.Float32,
+            ),
+            pl.col("nutriments")
+            .alias("arachidonic-acid_100g")
+            .map_elements(
+                lambda x: process_nutrients(
+                    nutrients=x, nutrient_name="arachidonic-acid"
+                ),
+                return_dtype=pl.Float32,
+            ),
+            pl.col("nutriments")
+            .alias("gamma-linolenic-acid_100g")
+            .map_elements(
+                lambda x: process_nutrients(
+                    nutrients=x, nutrient_name="gamma-linolenic-acid"
+                ),
+                return_dtype=pl.Float32,
+            ),
+            pl.col("nutriments")
+            .alias("dihomo-gamma-linolenic-acid_100g")
+            .map_elements(
+                lambda x: process_nutrients(
+                    nutrients=x, nutrient_name="dihomo-gamma-linolenic-acid"
+                ),
+                return_dtype=pl.Float32,
+            ),
+            pl.col("nutriments")
+            .alias("oleic-acid_100g")
+            .map_elements(
+                lambda x: process_nutrients(nutrients=x, nutrient_name="oleic-acid"),
+                return_dtype=pl.Float32,
+            ),
+            pl.col("nutriments")
```

Comment:
What happens if we add a new nutrient? It needs to be added manually in this file? Can we autodetect nutriments from the Parquet file, or from the  taxonomy?

Reply:
IMHO, the best solution would be to detect it from Parquet File as both exports (parquet and CSV) should be close to each other.

That said, this means that before creating each new field, we should alert CSV reusers of the future changes (isn't it @stephanegigandet ;-) ). A new nutriment is not an issue in the Parquet file, but it does in the CSV file.